src/hooks/useChartDataToCsv.ts

```diff
@@ -66,22 +111,53 @@ const getMapCsv = (series: any) => {
   return [headers, dataRows];
 };
 
+const getIsIntervalData = (series: any) => {
+  return (
+    series.filter((x: { confidence: boolean }) => x?.confidence === true)
+      .length > 0
+  );
+};
+
 const getChartCsv = (series: any, category: string, chartType: any) => {
   const isCompactBarData = chartType === "compact bar";
-  const seriesNames = series.map((s: any) => s.name);
+  const isIntervalData = getIsIntervalData(series);
```

Comment:
can you call `getIsIntervalData(series)` in the if statement and avoid setting the constant? genuine question, I don't know (you can in python).

please ignore if its some variation of "don't keep re-evaluating it"

Reply:
You can call it from within the if statement, I've just done it this way to be more explicit.